setup/sql/dbupdate_custom.php

```diff
@@ -0,0 +1,136 @@
+<#1>
+<?php
+// Migration Patch after apply this bugfix-> http://www.ilias.de/mantis/view.php?id=19795
+// Store the files provided for the users when they create feedback. (Peer-Feedback)
+if (!$ilDB->tableExists('exc_ass_file_order')) {
+	$fields = array(
+		'id' => array(
+			'type' => 'integer',
+			'length' => 4
+		),
+		'assignment_id' => array(
+			'type' => 'integer',
+			'length' => 4
+		),
+		'filename' => array(
+			'type' => 'text',
+			'length' => 100,
+			'notnull' => true
+		),
+		'order_nr' => array(
+			'type' => 'integer',
+			'length' => 4
+		)
+	);
+
+	$ilDB->createTable("exc_ass_file_order", $fields);
+	$ilDB->addPrimaryKey("exc_ass_file_order", array("id"));
+	$ilDB->createSequence("exc_ass_file_order", 1);
+}
```

Comment:
This belongs to the new feature development. I guess I can safely ignore this when merging.